sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/microsoft/azure/test/keyvault/KeyVaultIT.java

```diff
@@ -164,7 +163,9 @@ public void keyVaultWithVirtualMachineMSI() throws Exception {
             AZURE_KEYVAULT_URI,
             "app.jar"));
 
-        vm.runCommand(new RunCommandInput().withCommandId("RunShellScript").withScript(commands));
+        try (SSHShell sshShell = SSHShell.open(host, 22, VM_USER_USERNAME, VM_USER_PASSWORD)) {
```

Comment:
If `SSHShell.open(host, 22, VM_USER_USERNAME, VM_USER_PASSWORD)` execute failed, we will not know that, it's not good. Could you please add logic to retry.

If it still failed after few times retry, we should mark this unit test as `Failed` instead of ignore it.